recipes/packages.rb

```diff
@@ -118,35 +99,8 @@
                   'AIRFLOW_HOME' => node['airflow']['base_dir']})
     code <<-EOF
       set -e
-      #{node['conda']['base_dir']}/envs/airflow/bin/pip install --no-cache-dir apache-airflow["#{operator}"]==#{node['airflow']['version']}
+      #{node['conda']['base_dir']}/envs/airflow/bin/pip install apache-airflow["#{operator}"]==#{node['airflow']['version']} --constraint #{node['download_url']}/apache/airflow/1.10.10/requirements/requirements-python3.6.txt
```

Comment:
We have a script that helps us taking a snapshot of the artifacts so we can distribute them more easily. The script relies on the attributes defined in the attribute files.

So could you please move the building of the URL in `attributes/default.rb`?

Reply:
fixed